File: examples/node/src/00_connect.ts
import { createClient } from "./_mcp_sse_client.js";

async function main() {
  const client = createClient();
  await client.connect();

  console.log("Connected via SSE.");

  // Show tools (helps validate transport quickly)
  const tools = await client.listTools();
  console.log("\nTools:\n", JSON.stringify(tools, null, 2));

  client.close();
}

main().catch((e) => {
  console.error("ERROR:", e);
  process.exit(1);
});

File: examples/python/src/00_connect.py
import json
from client import McpSseClient, env

def main():
    c = McpSseClient(env("MCP_BASE_URL"))
    c.connect()
    print("Connected via SSE.")
    tools = c.list_tools()
    print(json.dumps(tools, indent=2))

if __name__ == "__main__":
    main()
